### ISSUE TITLE
COMP4010/5120 Teaching team review

### ISSUE THREAD

**Issue by tienvu95** [clarissdev/data-visualization-project-1#3]
It seems that there are many other variables which can describe the educational outcomes of young people in the dataset. Why did you select the variables that you propose? ( why highest_level_qualification_achieved_by_age_22_average_score, not highest_level_qualification_achieved_by_age_22_less_than_level_1, highest_level_qualification_achieved_by_age_22_level_1_to_level_2, etc..)

Do you think that question 2 can be answered by the output of question 1? Like income deprivation and job density can also be the factor that resulted in the differences between educational attainments of people in coastal and non-coastal regions. So if we you a legend to distinguish between coastas and non-coastal region in question 1, we may already answer both questions.

**Comment by clarissdev** [comment]
I agree that the two questions are quite similar, so I decided to change the question 1 as it is quite general. Therefore, no other variables are included XD.
